### PR TITLE
chore(atomic): fetch filetype/objecttype icons from coveo-styleguide package

### PR DESCRIPTION
**Pull request by ThibodeauJF**
https://coveord.atlassian.net/browse/KIT-935

### CODE REVIEW

File: packages/atomic/src/components/result-template-components/atomic-result-icon/atomic-result-icon.pcss [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
   > atomic-icon {
     width: 100%;
+    color: white;
```
Comment (samisayegh) [inline packages/atomic/src/components/result-template-components/atomic-result-icon/atomic-result-icon.pcss:8]: Why was this change needed?

**Review by samisayegh — APPROVED** [review]

File: packages/atomic/src/components/result-template-components/atomic-result-icon/atomic-result-icon.pcss (hunk 1 above)
Comment (ThibodeauJF) [reply packages/atomic/src/components/result-template-components/atomic-result-icon/atomic-result-icon.pcss:8]: The icons in our package (from JSUI) all had `fill="#fff"`, the ones from vapor don't have that attribute, so they've been replace by this property

File: packages/atomic/stencil.config.ts [hunk 2]
```diff
@@ -40,7 +40,13 @@ export const config: Config = {
     {
       type: 'dist',
       esmLoaderPath: '../loader',
-      copy: [{src: 'themes'}],
+      copy: [
+        {src: 'themes'},
+        {
+          src: '../node_modules/coveo-styleguide/resources/icons/svg/coveo-search-ui-filetypes',
```
Comment (olamothe) [inline packages/atomic/stencil.config.ts:46]: That works but it's not perfect.

When I look at react-vapor repo, these icons all seems to be fixed in times (16 months ago), and not imported/refreshed from Salesforce design system from time to time.

In an ideal world, I think we would actually want to depend from Salesforce design system directly, if possible.

**Review by olamothe — APPROVED** [review]

File: packages/atomic/stencil.config.ts (hunk 2 above)
Comment (ThibodeauJF) [reply packages/atomic/stencil.config.ts:46]: 🤔  I'll see what I can do
